src/OffsetProgressiveCurve.sol

```diff
@@ -90,17 +90,18 @@ contract OffsetProgressiveCurve is BaseCurve {
     /// @dev or to say that another way:
     /// $$\text{shares} = \sqrt{(s + o)^2 + \frac{2a}{m}} - (s + o)$$
     function previewDeposit(uint256 assets, uint256, /*totalAssets*/ uint256 totalShares)
-        public
+        external
         view
         override
         returns (uint256 shares)
     {
         require(assets > 0, "Asset amount must be greater than zero");
 
-        UD60x18 currentSupplyOfShares = UD60x18.wrap(totalShares).add(OFFSET);
```

Comment:
3.1 - Wrong Use of PRB-Math Library in Progressive Curves
We adjusted all of these to use `convert` instead of `wrap`.  This improved the accuracy of the curves, as well as the effect of the constructor arguments on the slope.